aries-core/src/js/components/core/Popover/Popover.js

```diff
@@ -0,0 +1,56 @@
+import { Box, Button, Drop } from 'grommet';
+import PropTypes from 'prop-types';
+import { Close } from 'grommet-icons';
+
+export const Popover = ({
+  title,
+  children,
+  footer,
+  targetRef,
+  onClose,
+  ...rest
+}) => {
+  return (
+    <Drop
+      round="xxsmall"
+      role="dialog"
+      elevation="medium"
+      stretch={false}
+      target={targetRef?.current}
+      onClose={onClose}
+      {...rest}
+    >
+      <Box gap="small" direction="row">
+        <Box flex gap="small" pad={{ vertical: 'small', left: 'small' }}>
+          {/* do we like this format of passing in
+           title children and footer? */}
+          {title}
+          {children}
+          {footer}
+        </Box>
+        {/* // see if we have a close t shirt size for the 8 px */}
+        <Box pad={{ right: 'small', top: '8px' }}>
+          <Button
+            size="small"
+            icon={<Close size="small" />}
+            onClick={onClose}
+            // should we make this a message that the user passes in?
+            // maybe we can have a default just incase they forget?
+            a11yTitle={`You are in a popover containing information,
```

Comment:
I think the user should be able to edit this

Reply:
you think we should have some default? or just have it be 
`a11Title={a11yTitle}`

Reply:
I think we could have a default set if they don't pass anything